### ISSUE TITLE
Ue Express generator for application file structure

### ISSUE THREAD

**Issue by Dreamwalker420** [AndrewKralovec/E-learn#6]
I believe we should use the Express generator for the file structure.

http://expressjs.com/en/starter/generator.html

**Comment by AndrewKralovec** [comment]
Yeah we can use it if you want. I guess it is more strucuted. My routes are all over the place. I don’t really like using jade. However i can develop in html. Then just use a html to jade converter. Ill upload what that will look like and if you like it we can do it that way.

**Comment by AndrewKralovec** [comment]
Made the switch
